tests/model_meta/tests.py

```diff
@@ -251,3 +254,19 @@ def test_relations_related_objects(self):
             ])
         )
         self.assertEqual([field.related_query_name() for field in AbstractPerson._meta._relation_tree], [])
+
+
+class ParentListTests(TestCase):
```

Comment:
2 optional items:
* TestCase could be SimpleTestCase or unittest.TestCase (but not sure it's worth the possibility that DB will be used later and not caught)
* assets could probably be single lines